### DISCUSSION TITLE
Not able to inject a span attribute using an extension to OT agent

### DISCUSSION THREAD

**Discussion by LavjeetGangwani** [open-telemetry/opentelemetry-java-instrumentation#3924] [Q&A]
I am trying to inject a span attribute by using the sample extension provided https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/examples/extension but seems like this sample extension is not working for me locally. I am using the same instrumentation point and at onMethodEnter I am accessing the current span and trying to inject a new attribute there. If someone can help me on this it would be amazing.

**Comment by mateuszrzeszutek** [comment]
Hi @LavjeetGangwani ,
Can you post the your code here? A repro repo/code would help us pinpointing what's going wrong here.
